phoenicis-apps/src/main/java/org/phoenicis/apps/MultipleRepository.java

```diff
@@ -78,6 +78,17 @@ public void addRepository(Repository repository) {
 		this.repositories = newRepositories;
 	}
 
+	public void addRepository(int index, Repository repository) {
+		Repository[] newRepositories = new Repository[this.repositories.length + 1];
+
+		System.arraycopy(this.repositories, 0, newRepositories, 1, index);
+		System.arraycopy(this.repositories, index, newRepositories, 1 + index, this.repositories.length - index);
+
+		newRepositories[index] = repository;
+
+		this.repositories = newRepositories;
```

Comment:
You're right, a list would be a better (easier) choice here. 
I think we should change the array in `MultipleRepository` to a list in a new PR, after this PR has been merged/closed.